__tests__/app.test.js

```diff
@@ -23,32 +42,51 @@ describe("/api/topics", () => {
   });
 });
 
-describe("Generic handler", () => {
-  test("GET: 404 sends an appropriate status and error message.", () => {
+describe("/api/articles", () => {
+  test("GET: 200 sends all articles in the correct format.", () => {
     return request(app)
-      .get("/api/top")
-      .expect(404)
+      .get("/api/articles")
+      .expect(200)
       .then((response) => {
-        expect(response.body.msg).toBe("Path not found.");
+        const { articles } = response.body;
+        expect(articles.length).toBe(13);
+        expect(Array.isArray(articles)).toBe(true);
+        articles.forEach((article) => {
+          expect(typeof article).toBe("object");
+        });
       });
   });
-});
-
-describe("/api", () => {
-  test("GET: 200 sends an object with all available endpoints.", () => {
+  test("Should return articles without a body, and with a commment_count.", () => {
     return request(app)
-      .get("/api")
+      .get("/api/articles")
       .expect(200)
       .then((response) => {
-        expect(typeof response.body).toBe("object");
+        const { articles } = response.body;
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("comment_count");
```

Comment:
We can be much more specific than this, there are other properties we expect to also be on the article object as well as _comment_count_ , check out `toMatchObject` for a jest matcher that is useful for this type of assertion 
And `expect.any(<dataType>)` for checking the values are any _string_ or _number_ and so on